pypeit/core/procimg.py

```diff
@@ -683,12 +690,26 @@ def subtract_overscan(rawframe, datasec_img, oscansec_img, method='savgol', para
             if var is not None:
                 _var[data_slice] = osvar
             continue
+            ossub = np.zeros_like(osfit)
+        elif method.lower() == 'odd_even':
+            ossub = np.zeros_like(osfit)
+            # Odd/even
+            if compress_axis == 1:
+                odd = np.median(overscan[:,1::2], axis=compress_axis)
+                even = np.median(overscan[:,0::2], axis=compress_axis)
+                # Do it
+                no_overscan[data_slice][:,1::2] -= odd[:,None]
+                no_overscan[data_slice][:,0::2] -= even[:,None]
+            else:
+                msgs.error('Not ready for this yet')
```

Comment:
Perhaps a slightly more information error message is needed - or just add "Contact a developer"

Reply:
Agreed!